Create-Vm.ps1

```diff
@@ -20,6 +20,9 @@ param
     [Parameter(Mandatory=$true, HelpMessage="The storage key for the storage account where custom images are stored")]
     [string] $StorageAccountKey,
 
+    [parameter(Mandatory=$true, HelpMessage="Public=separate IP Address, Shared=load balancers optimizes IP Addresses, Private=No public IP address.")]
+    [string] $LabIpConfig,
```

Comment:
Parameter for this script is passed in from parent script, but it originates from the CSV file at the user-level